projects/igniteui-angular/src/lib/icon/icon.references.ts

```diff
@@ -0,0 +1,586 @@
+    },
+});
+addIcon('first_page', {
+    default: {
+        name: 'first_page',
+        family: 'material',
+    },
+});
+addIcon('last_page', {
+    default: {
+        name: 'last_page',
+        family: 'material',
+    },
+});
+addIcon('add', {
+    default: {
+        name: 'add',
+        family: 'material',
+    },
+});
+addIcon('close', {
+    default: {
+        name: 'close',
+        family: 'material',
+    },
+});
+addIcon('error', {
+    default: {
+        name: 'error',
+        family: 'material',
+    },
+});
+addIcon('confirm', {
+    default: {
+        name: 'check',
+        family: 'material',
+    },
+});
+addIcon('cancel', {
+    default: {
+        name: 'close',
+        family: 'material',
+    },
+});
+addIcon('edit', {
+    default: {
+        name: 'edit',
+        family: 'material',
+    },
+});
+addIcon('delete', {
+    default: {
+        name: 'delete',
+        family: 'material',
+    },
+});
+addIcon('pin', {
+    default: {
+        name: 'pin-left',
+        family: 'imx-icons',
+    },
+});
+
+addIcon('unpin', {
+    default: {
+        name: 'unpin-left',
+        family: 'imx-icons',
+    },
+});
+addIcon('show', {
+    default: {
+        name: 'visibility',
+        family: 'material',
+    },
+});
+addIcon('hide', {
+    default: {
+        name: 'visibility_off',
+        family: 'material',
+    },
+});
+addIcon('tree_expand', {
+    default: {
+        name: 'chevron_right',
+        family: 'material',
+    },
+});
+addIcon('tree_collapse', {
+    default: {
+        name: 'expand_more',
+        family: 'material',
+    },
+});
+addIcon('chevron_right', {
+    default: {
+        name: 'chevron_right',
+        family: 'material',
+    },
+});
+addIcon('chevron_left', {
+    default: {
+        name: 'chevron_left',
+        family: 'material',
+    },
+});
+addIcon('expand_more', {
+    default: {
+        name: 'expand_more',
+        family: 'material',
+    },
+});
+addIcon('filter_list', {
+    default: {
+        name: 'filter_list',
+        family: 'material',
+    },
+});
+addIcon('import_export', {
+    default: {
+        name: 'import_export',
+        family: 'material',
+    },
+});
+addIcon('unfold_more', {
+    default: {
+        name: 'unfold_more',
+        family: 'material',
+    },
+});
+addIcon('unfold_less', {
+    default: {
+        name: 'unfold_less',
+        family: 'material',
+    },
+});
+addIcon('drag_indicator', {
+    default: {
+        name: 'drag_indicator',
+        family: 'material',
+    },
+});
+addIcon('group_work', {
+    default: {
+        name: 'group_work',
+        family: 'material',
+    },
+});
+addIcon('sort_asc', {
+    default: {
+        name: 'arrow_upward',
+        family: 'material',
+    },
+});
+addIcon('sort_desc', {
+    default: {
+        name: 'arrow_downward',
+        family: 'material',
+    },
+});
+addIcon('search', {
+    default: {
+        name: 'search',
+        family: 'material',
+    },
+});
+addIcon('sum', {
```

Comment:
AFAICT, this is only used for the Pivot and in both places I found it it was used to open dropdown to switch functions, so the alias name `functions` was better IMO.